### ISSUE TITLE
prevent excluded items from disappearing

### ISSUE THREAD

**Issue by matryer** [grafana/explore-logs#426]
This sounds counter intuitive, but when something is excluded (using the Exclude button) it currently disappears from under your mouse cursor. This happens because the filter is applied, and those values are no-longer in the results, so it disappears. Ideally, it would stay in-place, just showing the excluded toggle is active, so that people can go there to un-exclude it.

**Comment by gtk-grafana** [comment]
I was hoping there was an easy solution for this, but it comes down to how scenes/the app is interpolating variables, it doesn't look like there's an easy way to tell scenes to avoid interpolating a specific instance of a variable.

Ideally, we could skip interpolating other instances of existing variable filters when we're running the label values query, but since scenes grabs the current variable state behind the scenes, we'd have to manually interpolate.

I'll spend a bit more time poking into the scenes interpolate function to see if we can update core and maybe pass in an array of exclusions or something?

TL;DR This is hard
